geopyspark/geotrellis/__init__.py

```diff
@@ -258,20 +258,38 @@ def _asdict(self):
 GlobalLayout.__new__.__defaults__ = (256, None, 0.1)
 
 
-LocalLayout = namedtuple("LayoutDefinition", 'tile_size')
```

Comment:
My concern here is that exposing cols/rows is inconstant with `GlobalLayout` which can only receive tile size. These need to feel consistent and square tile in both cases is most common case.